views/eApostilles/additionalReference.ejs

```diff
@@ -14,7 +14,7 @@
             <div class="govuk-error-summary__body">
                 <ul class="govuk-list govuk-error-summary__list">
                     <li>
-                        <a href="#user-reference">Your reference must be 30 characters or fewer</a>
+                        <a href="#user-reference">Your reference is too long</a>
```

Comment:
Is the user meant to be able to "recover" from this? I'd argue this would be less useful to the user https://design-system.service.gov.uk/components/text-input/#if-the-number-is-too-high

Reply:
did this come up in UR? I think "summary" is misleading. typically the error in the summary will match the error for the input. https://design-system.service.gov.uk/components/error-summary

Reply:
Fair point, I will relay that back to LIz. I don't believe this was an issue in UR, this restriction is more to do with security and the copy change 🤷